src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/network/MixinPlayerListEntry.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of ViaFabricPlus - https://github.com/FlorianMichael/ViaFabricPlus
+ * Copyright (C) 2021-2024 FlorianMichael/EnZaXD <florian.michael07@gmail.com> and RK_01/RaphiMC
+ * Copyright (C) 2023-2024 contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.florianmichael.viafabricplus.injection.mixin.fixes.minecraft.network;
+
+import de.florianmichael.viafabricplus.injection.access.IPlayerListEntry;
+import net.minecraft.client.network.PlayerListEntry;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+
+@Mixin(PlayerListEntry.class)
+public abstract class MixinPlayerListEntry implements IPlayerListEntry {
+
+    @Unique
+    private static int viaFabricPlus$GLOBAL_INDEX = 0;
```

Comment:
I believe this should be reset when disconnecting from the server? In this case I suggest moving it into the ClientsideFixes class and use one of the located events in there

Reply:
it can be reset, or can be ignored
only case it becomes an issue is if you were to receive over integer limit amount of player list entries without restarting your client which seems improbable, but yeah i can move it

Reply:
Yeah move it to ClientsideFixes and reset it, so the code is safe.